client/yarn.lock

```diff
@@ -0,0 +1,10849 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
```

Comment:
So, we're using yarn right from the start?

Reply:
I will probably remove yarn.